app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -1166,4 +1166,21 @@ public function getIncrementIdFromTransaction( $transaction ) {
             ? explode("|", $transaction->order->cart->display_id)[0]
             : $transaction->order->cart->display_id;
     }
-}
\ No newline at end of file
+
+    /**
+     * Generate (if) secure url by route and parameters
+     *
+     * @param   string $route
+     * @param   array $params
+     * @return  string
+     */
+    public function getSecureUrl($route = '', $params = array()){
```

Comment:
I suggest you name this simply `getUrl` or `getMagentoUrl`. It's confusing if it is named secure but can return unsecure url.

Reply:
Also suggest we don't add `$params` arg if we're not using today. YAGNI.

Reply:
Will fix the naming.